pkg/controller/servicebindingrequest/binder.go

```diff
@@ -501,48 +502,46 @@ func (b *Binder) update(objs *unstructured.UnstructuredList) ([]*unstructured.Un
 		}
 
 		log.Debug("Updating object...")
-		if err := b.client.Update(b.ctx, updatedObj); err != nil {
-			return nil, err
-		}
+		gvr, _ := meta.UnsafeGuessKindToResource(updatedObj.GroupVersionKind())
```

Comment:
It should be possible to propagate the Reconciler's `RestMapper` field here and use instead of `UnsafeGuessKindToResource`.

Reply:
Good point. I will change it accordingly.